plugins/auth-backend/src/identity/types.ts

```diff
@@ -28,13 +28,19 @@ export interface AnyJWK extends Record<string, string> {
  * @public
  */
 export type TokenParams = {
-  /** The claims that will be embedded within the token */
+  /**
+   * The claims that will be embedded within the token. At a minimum, this should include
+   * the subject claim, `sub`. It is common to also list entity ownership relations in the
+   * `ent` list. Additional claims may also be added at the developer's discretion except
+   * for the following list, which will be overwritten by the TokenIssuer: `iss`, `aud`,
+   * `iat`, and `exp`.
```

Comment:
Could we add a note here that any additional claims are always potentially breaking? basically we reserve the right to add new claims in the future without calling it a breaking change.

Reply:
Absolutely. Should that be phrased "Backstage maintains...", "The Backstage authors maintain...", or "Spotify maintains..."?